docs/download/guide.md

```diff
@@ -35,9 +35,9 @@ There are two prerequesites to using the Score Client: DACO Cloud Access status
 
 ### DACO Cloud Access
 
-DACO Cloud Access is prerequisite to using the Score Client. To apply for DACO access please follow the instructions provided at [https://icgc.org/daco](https://icgc.org/daco). Once approved, you will be able to [login](https://dcc.icgc.org/) to the Data Portal to generate an [Access Token](#access-tokens). To login, click on the “Login” link in the upper right-hand corner of the page. When prompted, choose to login with either your [ICGC.org login](https://icgc.org/user/login) or one of the supported OpenID providers (e.g. Google). After successful authentication, you will know that you have Cloud Access to the controlled tier if the “login” link is replaced with a green cloud icon:
+DACO Cloud Access is prerequisite to using the Score Client. To apply for DACO access please follow the instructions provided at [https://daco.icgc-argo.org/](https://daco.icgc-argo.org/). Once approved, you will be able to [login](https://dcc.icgc.org/) to the Data Portal to generate an [Access Token](#access-tokens). To login, click on the “Login” link in the upper right-hand corner of the page. When prompted, choose to login with your ICGC DACO-approved account. After successful authentication, you will know that you have Cloud Access to the controlled tier if the “login” link is replaced with a green cloud icon:
```

Comment:
@hknahal on this page at the top under **Process at a Glance**

maybe we should just say `Apply for DACO Access if not already approved` instead of `Apply for DACO Cloud Access if not already approved` 
Thoughts?

Reply:
also under `Access Tokens` let's change 
...`will be available after acquiring DACO Cloud Access `

to `will be available after acquiring DACO Access`